examples/grpc_server/src/SeatsServiceImpl.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef VELOCITAS_SERVICE_IMPL_Seats_H
+#define VELOCITAS_SERVICE_IMPL_Seats_H
+
+#include "vehicle/Vehicle.hpp"
```

Comment:
Off course using same includes as in https://github.com/eclipse-velocitas/vehicle-app-cpp-template/blob/main/app/src/SampleApp.h#L22 does not work when building for CI.

But in this file they refer to `"vehicle/Vehicle.h"` instead:

https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/examples/set-data-points/src/SetDataPointsApp.cpp#L23

@MP91 @BjoernAtBosch - when shall one use `Vehicle.h`and when `Vehicle.hpp`?



```

In file included from /__w/vehicle-app-cpp-sdk/vehicle-app-cpp-sdk/examples/grpc_server/src/Launcher.cpp:1:
/__w/vehicle-app-cpp-sdk/vehicle-app-cpp-sdk/examples/grpc_server/src/SeatsServiceImpl.h:20:10: fatal error: vehicle/Vehicle.hpp: No such file or directory
   20 | #include "vehicle/Vehicle.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
Or is it so that the CI is actually cheating and does not even read the AppManifest.json? I.e. that it works for `vehicle.h`is because of the old generated model at https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/examples/example_model/vehicle/Vehicle.h? 

Seems to be like that and then we are quite screwed anyway. If so exclude the new grpc examples from CI and add an issue that we better should not rely on a handwritten `Vehicle.h`, or? Because I assume we do not want to add handwritten grpc files as well, right?

Reply:
As far as I can see, the sdk currently is not using the new generated model at all. So yes the only way currently is to use the custom model in the examples...

Reply:
Which would be a quite ugly work-around, and we would then need add code generated from the gRPC proto file as well, so I will most likely opt for not compiling the code of the new examples as of now.